orion-ui/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "@prefecthq/orion-design": "1.1.6",
-    "@prefecthq/prefect-design": "1.1.29",
+    "@prefecthq/prefect-design": "^1.1.32",
```

Comment:
@nicolascribbles Can you please run `npm i @prefecthq/prefect-design@latest --save-exact` and push that change? The `^` shouldn't be here since its a direct dependency. We only use the `^` for peer dependencies.